src/FactorAPI/ReadResponse.java

```diff
@@ -1,7 +1,9 @@
 package factorapi;
 
+import java.util.List; 
+
 public interface ReadResponse {

```

Comment:
The ReadResponse interface should define a method, like getData(), for retrieving response data as a list of integers.